README.md

```diff
@@ -180,10 +180,11 @@ The main people in charge of this repo are:
 
 ## Contribute
 
-To contribute, please use github [issues](https://github.com/flyntwp/flynt-starter-theme/issues). Pull requests are accepted.
+To contribute, please use GitHub [issues](https://github.com/flyntwp/flynt-starter-theme/issues). Pull requests are accepted. Please also take a moment to read the [contributing guidelines](https://github.com/flyntwp/guidelines/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/flyntwp/guidelines/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
it's called Code of Conduct in the file, but lowercase code of conduct here. Which one do we go with?